src/lsm/scan_builder.zig

```diff
@@ -18,14 +19,27 @@ const TimestampRange = @import("timestamp_range.zig").TimestampRange;
 
 const Error = @import("scan_buffer.zig").Error;
 
-/// ScanBuilder is a helper to create and combine scans using
-/// any of the Groove's secondary indexes.
 pub fn ScanBuilderType(
```

Comment:
Maybe just "ScanBuilder is a helper to create and combine scans using any of the Groove's indexes."?

Reply:
:+1:  I removed the comment while I was thinking in the `TODO` next line, since it won't be single-groove soon.